base/Makefile

```diff
@@ -249,50 +248,16 @@ ${CLUSTER_GEN}/remote-healthcheck-image: ${CLUSTER_GEN}/healthcheck-image-tagged
 	@  exit 1
 	@fi
 	kubectl config use-context "kctf_${PROJECT}_${ZONE}_${CLUSTER_NAME}" >&2
+	kubectl config set-context --current --namespace="${CHALLENGE_NAME}"
```

Comment:
I'm setting the namespace during config load. This has the nice side effect that kctf-kubectl still works without specifying a namespace since it defaults to the last challenge that you touched.

Reply:
We probably need to update the troubleshooting playbook then. How do you switch to another task?

Reply:
either allways set --namespace=challengename, run the config set-context command manually, or run any make command in the challenge dir, e.g. make status.
What would you recommend?

Reply:
--namespace=challengename is probably ok :) similar to before